src/main/java/com/acme/dbo/txlog/Facade.java

```diff
@@ -1,54 +1,52 @@
 package com.acme.dbo.txlog;
 
-import javafx.util.Pair;
-
-import java.util.Arrays;
-
-import static java.lang.Math.abs;
-
 public class Facade {
-    private static Boolean isDecorated=true;
 
     public static void log(int message) {
-    Controller.logInteger(message,  isDecorated);
+        Controller.intCommand = new IntCommand(message);
```

Comment:
It's time to make Controller great again - make it true object ))) By replacing _static_ methods.